pages/content/amp-dev/documentation/components/reference/amp-lightbox-v0.1@ru.md

```diff
@@ -91,7 +91,7 @@ teaser:
         <td width="40%"><strong>scrollable (необязательно)</strong></td>
         <td>Наличие атрибута <code>scrollable</code> означает, что содержимое окна просмотра можно прокручивать, если оно выходит за рамки окна.
           <br><br>
-            <strong>Примечание.</strong> Атрибут <code>scrollable</code> нельзя использовать с компонентом <code><amp-lightbox></code> в объявлениях HTML с технологией AMP. Подробную информацию можно найти в <a href="#a4a">этом разделе</a>.</td>
+            <strong>Примечание.</strong> Атрибут `scrollable` нельзя использовать с компонентом `&lt;amp-lightbox&gt;` в объявлениях HTML с технологией AMP. Подробную информацию можно найти в <a href="#a4a">этом разделе</a>.</td>
```

Comment:
Hm... this only works via one of our markdown extensions but wouldn't in other environments than Grow. We should therefore probably stick to the default markdown syntax which doesn't allow markdown in HTML elements. So this would become:

```suggestion
            <strong>Примечание.</strong> Атрибут <code>scrollable</code> нельзя использовать с компонентом <code>&lt;amp-lightbox&gt;</code> в объявлениях HTML с технологией AMP. Подробную информацию можно найти в <a href="#a4a">этом разделе</a>.</td>
```